### PR TITLE
I define the interface Operations

### PR DESCRIPTION
**Pull request by volocarucatalin**
- create and implement interface Operations
-  change way of reading the matrix from file

### CODE REVIEW

File: src/main/java/catalin/Main.java [hunk 1]
```diff
@@ -7,20 +8,14 @@
...
-        Reader file1 = new Reader(fileMatrix1);
-        Reader file2 = new Reader(fileMatrix2);
-        Matrix operation = new Matrix();
-        matrixOne = file1.readMatrices(matrixOne);
-        matrixTwo = file2.readMatrices(matrixTwo);
-
-        matrixAddition = operation.sum(matrixOne,matrixTwo);
-        matrixMultiply = operation.multiplication(matrixOne,matrixTwo);
-
+        Matrix matrix1 = new Reader().readMatrices(fileMatrix1);
```
Comment (Sergiu221) [inline src/main/java/catalin/Main.java]: Please don't create every time a new object just define readMatrix() as a static method,

File: src/main/java/catalin/Matrix.java [hunk 2]
```diff
@@ -1,33 +1,37 @@
...
-                int a;
-                int b = matrix1[i][j];
-                int c = matrix2[i][j];
-                a = (b * c);
-                matrixMultiplication[i][j] = a;
+    public Matrix(int lines, int columns, int[][] body) {
+        this.body = body;
+    }
+
+    public Matrix multiplication(Matrix matrix1) {
```
Comment (Sergiu221) [inline src/main/java/catalin/Matrix.java]: Please remove 1 from matrix1.

File: src/main/java/catalin/Reader.java [hunk 3]
```diff
@@ -1,27 +1,23 @@
 package catalin;
 
 import java.io.*;
+import java.util.regex.MatchResult;
 
 public class Reader {
-    File file;
-    BufferedReader bufferedReader ;
+    File filename;
```
Comment (Sergiu221) [inline src/main/java/catalin/Reader.java]: Please remove all the fields from Reader.java and use local variables for each function.

File: src/main/java/catalin/Operations.java [hunk 4]
```diff
@@ -0,0 +1,6 @@
+package catalin;
+
+public interface Operations {
+    Matrix sum(Matrix matrix1);
```
Comment (Sergiu221) [inline src/main/java/catalin/Operations.java]: Also here remove 1.

**Comment by volocarucatalin** [comment]
I have done the request from above

**Comment by Sergiu221** [comment]
This PR it look fine.

**Review by Sergiu221 — COMMENTED** [review]
This PR look fine.

**Review by Sergiu221 — COMMENTED** [review]
This PR look fine.

**Review by Sergiu221 — APPROVED** [review]

**Pull request merged**